src/__experimental__/components/fieldset/__snapshots__/fieldset.spec.js.snap

```diff
@@ -3,6 +3,356 @@
 exports[`Fieldset renders correctly 1`] = `
 <styled.fieldset
   data-component="fieldset"
+  m={0}
```

Comment:
Theme should not be rendered here

in `fieldset.spec.js` do that:

import { noThemeSnapshot } from "../__spec_helper__/enzyme-snapshot-helper"; (edited) expect(noThemeSnapshot(wrapper)).toMatchSnapshot();